system-int-test/test/cmr/system_int_test/search/collection_relevancy/collection_temporal_relevancy.clj

```diff
@@ -82,4 +83,17 @@
       [coll3 coll1 coll2 coll5 coll4]
 
       "Date range including collection with early ranges"
-      ["1955-01-01T10:00:00Z,1999-03-01T0:00:00Z"] [coll5 coll6 coll2])))
+      ["1955-01-01T10:00:00Z,1999-03-01T0:00:00Z"] [coll5 coll6 coll2])
+
+   (testing "Keyword search, sort by end date"
+     (is (d/refs-match-order? [coll3 coll4 coll1 coll2 coll5 coll6]
+                              (search/find-refs :collection {:keyword "coll"})))
+
+     (testing "Multiple ongoing collections"
+       (let [coll7 (d/ingest "PROV1" (dc/collection {:entry-title "coll 7"
```

Comment:
Yeah I'll add another test. It's "discouraged" to have an end date in the future, it's an ARC thing. I think for now we just sort them and if someone has an end date in the future, it'll show up first. If we really want to combat it, the options are to set :missing to a date way in the future or write a script that sorts nil first, which i think is overkill.